src/Components/Scenes/Chrome/Search/Search-Container.js

```diff
@@ -3,21 +3,24 @@ import Search from './Search';
 import {
   searchHandleInput,
   searchResultRequest,
-  handleKeyPress
+  handleKeyPress,
+  handleSearchClose
 } from '../../../../Actions/searchResultActions';
 
 const mapStateToProps = ({ searchResult }) => {
   return {
     searchInput: searchResult.searchInput,
-    isSearchClicked: searchResult.isSearchClicked
+    isSearchClicked: searchResult.isSearchClicked,
+    closeModal: searchResult.handleSearchClose
```

Comment:
I don think this should be here. You are not using closeModal in component.
and there is not handleSearchClose in initialState

Reply:
Good catch I removed that from mapStateToProps.